docs/cn/instructions/eventmesh-runtime-quickstart.zh-CN.md

```diff
@@ -65,17 +65,7 @@ sh start.sh
 
 > 注：插件模块遵循java spi机制，需要在对应模块中的/main/resources/META-INF/services 下配置相关接口与实现类的映射文件
 
-**2.3.2 配置VM启动参数**
-
-```java
--Dlog4j.configurationFile=eventmesh-runtime/conf/log4j2.xml
--Deventmesh.log.home=eventmesh-runtime/logs
--Deventmesh.home=eventmesh-runtime
--DconfPath=eventmesh-runtime/conf
-```
-> 注：如果操作系统为Windows, 可能需要将文件分隔符换成\
-
-**2.3.3 配置build.gradle文件**
+**2.3.2 配置build.gradle文件**
```

Comment:
The configuration process of plugin has changed, we no longer need to modify the `build.gradle` file and compile again, you can refer to #421.

Reply:
yes, I got it.